src/js/components/awardv2/idv/IdvContent.jsx

```diff
@@ -46,6 +48,10 @@ export default class IdvContent extends React.Component {
                     <IdvDates
                         dates={this.props.overview.dates} />
                 </div>
+                <div className="award__agencyAdditional" id="award-additional-information">
```

Comment:
Class names should be delineated with dashes (`agency-additional`) instead of camelCase